qemu/tests/data_plane_machine_type.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from autotest.client.shared import error
+
+from virttest import utils_misc
+from virttest import env_process
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    boot with different machine type:
+    1) get supported machine type
+    2) boot guest with different machine types
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error.context("Get host's supported machine type", logging.info)
+    qemu_binary = utils_misc.get_qemu_binary(params)
+    tmp = utils_misc.get_support_machine_type(qemu_binary)
+    (supported_machine_types, expected_system_versions) = tmp
```

Comment:
better put these two lines into one line.
if you don't need `expected_system_versions`, you can do it like this:
`supported_machine_types, _ = utils_misc.get_qemu_binary(params)`